refresh/index.js

```diff
@@ -1087,6 +1087,7 @@ module.exports = Generator.extend({
           {
             appName: this.projectName,
             executableName: this.executableModule,
+            chartName: helpers.sanitizeAppName(this.bluemix.name),
```

Comment:
As best I can tell, the chartName property is not used.  Am I looking in the wrong place?

https://github.com/ibm-developer/generator-ibm-cloud-enablement/blob/65b3a691ed06fdedca5705b12fcbbfea9642c767/generators/kubernetes/index.js#L93

Reply:
It's used in the README under the Kubernetes heading :-)